sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -247,6 +262,7 @@ export class MessageReceiver extends LinkEntity {
       audience: `${context.namespace.config.endpoint}${context.entityPath}`
     });
     if (!options) options = {};
+    this._retryOptions = options.retryOptions || {};
```

Comment:
Is the '|| {}' correct? Where does the "no values were specified, default to 'x'" happen?

Reply:
Also, I assume they can specify one value (say max retries) but not another and we should provide a default for the unspecified value, correct?

Reply:
`retry()` method from core-amqp validates the RetryConfig that we pass. We don't need to add additional validations.